packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -450,6 +437,66 @@ export default class VTable {
         }
         return result;
     }
+
+    private setEmptyTableCells() {

```

Comment:
Normally a function named "set***" should take some parameter to set. But here it doesn't require parameter. So use a better name which can reflect its functionality.